source/transactions/transactions.rst

```diff
@@ -810,26 +810,30 @@ certain retries of commitTransaction and abortTransaction) to the same mongos.
 When to unpin
 ^^^^^^^^^^^^^
 
-Drivers MUST unpin a ClientSession when a command within a transaction,
-including commitTransaction and abortTransaction, fails with a
-TransientTransactionError. Transient errors indicate that the transaction
-in question has already been aborted or that the pinned mongos is
+Drivers MUST unpin a ClientSession in the following situations:
+
+#. The transaction is aborted. The session MUST be unpinned regardless of
+whether or the abort succeeds or fails. If the operation fails with a
```

Comment:
`the abort` -> `the abortTransaction command`.

Reply:
This might have been intentional, as I [previously asked](https://github.com/mongodb/specifications/pull/968#discussion_r617933562) about the case where no command is executed (e.g. abort immediately after starting a transaction). Perhaps revise to say "regardless of whether the `abortTransaction` command succeeds or fails, or was executed at all"?

Reply:
Oh I see. The current wording or your suggestion SGTM in that case.

Reply:
Revised.